mod/file/thumbnail.php

```diff
@@ -6,7 +6,12 @@
  */
 
 // Get engine
-require_once __DIR__ . '/../../vendor/autoload.php';
+$initialRoot = dirname(dirname(__DIR__));
+$backupRoot = dirname(dirname(dirname($initialRoot)));
+
+if (!include_once "$initialRoot/vendor/autoload.php") {
```

Comment:
Or maybe not because the plugins are symlinked in... Anyway it would be nice to use something like this that doesn't introduce global vars:
```php
call_user_func(function () {
    // determine $path
    require $path;
});
```